src/modules/doubt/doubt.controller.ts

```diff
@@ -33,8 +34,8 @@ export class DoubtController {
   @Get()
   @ApiOkResponse(responsedoc.getAllDoubts)
   @ApiOperation({ summary: 'Retrieve doubts list' })
-  async getAllDoubts() {
-    return await this.doubtService.getAllDoubts();
+  async getAllDoubts(@Query('skipNum') skipNum: string) {
```

Comment:
same here.

Reply:
for ease of writing query from client side
it is then getting parsed into an integer of base 10